core/server/api/themes.js

```diff
@@ -65,6 +65,11 @@ themes = {
                 // We use the model, not the API here, as we don't want to trigger permissions
                 return settingsModel.edit(newSettings, options);
             })
+            // Load theme translations when activating a theme
+            // (when starting Ghost, or when switching themes).
+            .then(function () {
+                return common.i18n.loadThemeTranslations();
```

Comment:
This can be triggered via event listeners as well 🙃

Reply:
Event listener to reload theme translations for active theme change done.